index.ts

```diff
@@ -13,11 +13,20 @@ import { Request as JWTRequest } from "express-jwt";
 
 dotenvExpand.expand(dotenv.config());
 
-import { jwtAuthz } from "@aserto/aserto-node";
+import { Authorizer, Middleware, getSSLCredentials } from "@aserto/aserto-node";
 import { getConfig } from "./config";
 
 const authzOptions = getConfig();
 
+const ssl = getSSLCredentials(authzOptions.authorizerCertCAFile);
```

Comment:
This seems to assume Topaz. What if the config file points to the hosted environment?  Seems like we want to make the "getSSLCredentials" conditional on whether this was passed in?

Reply:
Good catch, I updated the code to work also for hosted.

Reply:
I wonder if @aserto/aserto-node ought to provide a way to return `credentials.createSsl()` for an empty CA File argument to `getSSLCredentials` - this will make the consumer code self-contained without any need to be aware of the "@grpc/grpcjs" package.  Thoughts?